src/main/java/Onandon.java

```diff
@@ -0,0 +1,211 @@
+
+import Onandon.module.*;
+import Onandon.print.Print;
+import Onandon.exception.OnandonEmptyException;
+import Onandon.exception.OnandonUnknownException;
+import Onandon.exception.OnandonNotaskException;
+import Onandon.checkpoint.Checkpoint;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class Onandon {
+    static ArrayList<Task> tasks = new ArrayList<Task>(100);
+
+    public static int printTodoAction(String inputText, ArrayList<Task> tasks, int cnt){
+        String description;
+        description = inputText.replaceAll("todo ", "");
+
+        Print.printFormAbove();
+        tasks.add(new Todo(description));
+        System.out.println("\t\t" + tasks.get(cnt).toString());
+        cnt++;
+        Print.printCnt(cnt);
+        return cnt;
+    }
+
+    public static int printDeadlineAction(String inputText, ArrayList<Task> tasks, int cnt){
+        String description;
+        String by;
+        int indexBy;
+
+        indexBy = inputText.indexOf("/by");
```

Comment:
Try to refactor magic literals into constants to improve readability. Similar issue in other places.